### ISSUE TITLE
Gamepad 8BitDo M30 does not connect to Batocera

### ISSUE THREAD

**Issue by AlexLeoni** [rg35xx-cfw/rg35xx-cfw.github.io#133]
Good afternoon, is it possible to add support for the 8BitDo M30 gamepad to Batocera? Doesn't connect :(

**Comment by TrilIiard** [comment]
Same for the 8BitDo Pro 2

**Comment by TrilIiard** [comment]
Yes, its showing "connecting to 8BitDo Pro 2" then "trusting 8BitDo Pro 2" and then "connection failed"

**Comment by DutchmanAZ** [comment]
After the initial pairing, even when it says failed, try rebooting. Once you do that, don't try to re-pair it. Just turn it on. it usually just pairs right up no problem using an 8bitdo Lite 2. 

I think the Bluetooth in these units isn't the best.

**Comment by TrilIiard** [comment]
Correct, that works. But it works fine on Stock OS with two Controllers at the same time so the Hardware is not the problem here.

**Comment by AlexLeoni** [comment]
On the stock firmware 8BitDo M30 connects and works perfectly, without errors.

**Comment by AlexLeoni** [comment]
> After the initial pairing, even when it says failed, try rebooting. Once you do that, don't try to re-pair it. Just turn it on. it usually just pairs right up no problem using an 8bitdo Lite 2.

This connection option doesn't work :(
